sql/log_event.cc

```diff
@@ -2276,7 +2289,9 @@ log_event_print_value(IO_CACHE *file, const uchar *ptr,
       struct timeval tm;
       my_timestamp_from_binary(&tm, ptr, meta);
       int buflen= my_timeval_to_str(&tm, buf, meta);
-      my_b_write(file, buf, buflen);
+      MY_ATTRIBUTE((unused)) int write_res=
+        my_b_write(file, reinterpret_cast<const uchar*>(buf), buflen);
+      DBUG_ASSERT(write_res == 0);
```

Comment:
Looking at default and MYSQL_TYPE_ENUM branches in the switch, write_res can be checked on release build and zero returned if !write_res

Reply:
Agree. Fixed.